src/app/GitExtensions.Extensibility/Translations/Xliff/TranslationUtil.cs

```diff
@@ -432,6 +437,14 @@ private static IEnumerable<Type> GetLoadableTypes(Assembly assembly)
         }
     }
 
+    private static HashSet<string>? GetLocalizablePropertiesFromAttribute(object item)
+    {
+        return item.GetType()
+            .GetCustomAttribute<LocalizablePropertiesAttribute>()
+            ?.TranslatableProperties
+            ?.ToHashSet(StringComparer.Ordinal);
```

Comment:
Since the `HashSet` is re-created every time, it cannot speedup `.Contains`.
Rather change the type of `LocalizablePropertiesAttribute.TranslatableProperties`. Or else if impossible, search in the `string[]` directly.